packages/idempotency/package.json

```diff
@@ -80,10 +80,25 @@
   },
   "dependencies": {
     "@aws-lambda-powertools/commons": "^1.12.1",
-    "@aws-sdk/lib-dynamodb": "^3.231.0",
     "@aws-sdk/util-base64-node": "^3.209.0",
     "jmespath": "^0.16.0"
   },
+  "peerDependencies": {
```

Comment:
This is great, we should replicate it to other utilities.

Reply:
Yep, I'll do it in the next PR